specification/hdinsight/data-plane/Microsoft.HDInsight/preview/2018-11-01-preview/livySpark.json

```diff
@@ -0,0 +1,1128 @@
+            "required": false,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkSessionCollection"
+            }
+          }
+        }
+      },
+      "post": {
+        "tags": [
+          "SparkSession"
+        ],
+        "description": "Create a new spark session.",
+        "operationId": "SparkSession_Create",
+        "x-ms-examples": {
+          "Create a new spark session": {
+            "$ref": "./examples/HDI_SparkSession_Create.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/requestedByParameter"
+          },
+          {
+            "name": "sparkSessionJobRequest",
+            "in": "body",
+            "description": "Livy compatible session job request payload.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/SparkSessionJobRequest"
+            }
+          }
+        ],
+        "responses": {
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/SparkSessionJob"
+            }
+          }
+        }
+      }
+    },
+    "/livy/sessions/{sessionId}": {
+      "get": {
+        "tags": [
+          "SparkSession"
+        ],
+        "description": "Gets a single spark session.",
+        "operationId": "SparkSession_Get",
+        "x-ms-examples": {
+          "Gets a single spark session": {
+            "$ref": "./examples/HDI_SparkSession_Get.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "sessionId",
+            "in": "path",
+            "description": "Identifier for the session.",
+            "required": true,
+            "type": "integer",
+            "format": "int32"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Success",
+            "schema": {
+              "$ref": "#/definitions/SparkSessionJob"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "SparkSession"
+        ],
+        "description": "Cancels a running spark session.",
+        "operationId": "SparkSession_Delete",
+        "x-ms-examples": {
+          "Cancels a running spark session": {
+            "$ref": "./examples/HDI_SparkSession_Delete.json"
+          }
+        },
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
```

Comment:
You declare a no-schema response, means there is nothing produced and we shall not see `produces` then

Reply:
Fixed it. This request should contain the response message in the body of the response. Added schema for it.